packages/webview_flutter/example/pubspec.yaml

```diff
@@ -5,6 +5,7 @@ version: 1.0.1
 
 environment:
   sdk: ">=2.0.0-dev.68.0 <3.0.0"
+  flutter: ">=1.9.1+hotfix.2 <2.0.0"
```

Comment:
My local branch is out of date, I just meant to pick the most recent stable. Changed it to `.4`.

I think so, especially since we'll need the v2 Registrant.